stories/Atom/BaseTypography/BaseTypography.stories.mdx

```diff
@@ -695,19 +695,19 @@ export const getCaptionForLocaleList = (locale) => {
       const engText = {
         data: [
           {
-            label: "United Nations Development Programme",
+            label: "Nations Office for Disaster Risk Reduction",
             text:
-              "The United Nations Development Programme is the global development network of the United Nations. It promotes technical and investment cooperation among nations and advocates for change and connects countries to knowledge, experience and resources to help people build a better life for themselves.",
+              "UNDRR brings governments, partners and communities together to reduce disaster risk and losses to ensure a safer, more sustainable future.",
           },
           {
-            label: "United Nations Development Programme",
+            label: "Nations Office for Disaster Risk Reduction",
```

Comment:
Some find/replaces went off, resulting in "Nations Office for Disaster Risk Reduction"

Reply:
oh okay. will rectify that.